projects/models.py

```diff
@@ -11,4 +12,4 @@ class Post(models.Model):
     slug = models.SlugField(blank=True, unique=True)
 
     def __str__(self):
-        return self.title
\ No newline at end of file
+        return self.title
```

Comment:
`__str__` sets the value of `{{ rand_post }}` in the template :)
(in this case, value will be the title)